data-pipeline/src/trace_exporter/mod.rs

```diff
@@ -522,6 +526,10 @@ impl TraceExporter {
                     }
                     match response.into_body().collect().await {
                         Ok(body) => {
+                            info!(
```

Comment:
Shouldn't this be a debug log?

Reply:
that's a name-value pair, could be shorthand

https://docs.rs/tracing/latest/tracing/#recording-fields